doc/dynamo0p3.rst

```diff
@@ -966,11 +981,13 @@ and ``ncell_3d`` scalar arguments. The full set of rules are then:
        intent ``in``.
     2) Include the number of cells in the 2D mesh, ``ncell_2d``, which is
        an integer with intent ``in``.
-    3) For each (CMA operator) argument specifed in meta-data:
+    3) For each CMA operator argument specifed in meta-data:
 
        1) Include it and its associated parameters (see Rule 5 of CMA
 	  Assembly kernels).
 
+    4) For each scalar argument specified in meta-data include the
```

Comment:
Well spotted. The code is right, the documentation is wrong - I've corrected it (bc5376c).